ecs/convert.go

```diff
@@ -562,6 +562,11 @@ func gpuRequirements(s types.ServiceConfig) int64 {
 					return resource.DiscreteResourceSpec.Value
 				}
 			}
+			for _, device := range reservations.Devices {
+				if len(device.Capabilities) == 1 && device.Capabilities[0] == "gpu" {
```

Comment:
Do we want to restrict the capabilities to "gpu" only? Could we be more flexible? I thought these flags are passed to a Docker Engine and the engine can return an error if we request a capability that is not supported.

Reply:
we don't target docker engine here but AWS EC2 and need to select machine type based on number of requested GPUs